fitbenchmarking/fitbenchmarking
# 620 pp docs

#### Description of Work

Fixes #620 


#### Testing Instructions

1. Check the docs display ok
2. Check the wording
3.

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
